frontend/src/container/ResourceAttributesFilter/ResourceAttributesFilter.tsx

```diff
@@ -22,60 +29,129 @@ function ResourceAttributesFilter({
 		handleClearAll,
 		handleFocus,
 		handleChange,
+		handleEnvironmentChange,
 		selectedQuery,
 		optionsData,
 		loading,
 	} = useResourceAttribute();
 
+	const [environments, setEnvironments] = useState<
+		SelectOption<string, string>[]
+	>([]);
+
+	const [selectedEnvironments, setSelectedEnvironments] = useState<string[]>([]);
+
+	const queriesExcludingEnvironment = useMemo(
+		() =>
+			queries.filter(
+				(query) => query.tagKey !== 'resource_deployment_environment',
```

Comment:
NIT: `resource_deployment_environment` to a contant.

Reply:
Will pick this in another PR